src/components/CertificateSettings.vue

```diff
@@ -39,7 +39,11 @@
 			label="label"
 			track-by="id"
 			@select="selectCertificate" />
-		<Button type="primary" :disabled="certificate === null" @click="updateSmimeCertificate">
+		<Button
+			type="primary"
+			:disabled="certificate === null"
+			aria-label="Update Certificate"
+			@click="updateSmimeCertificate">
 			Update Certificate
```

Comment:
unrelated to your change but we are lacking translation here

Reply:
:heavy_check_mark: